examples/v1/snap-plugin-collector-rand-streaming/rand/rand.go

```diff
@@ -13,7 +13,7 @@
     See the License for the specific language governing permissions and
     limitations under the License.
 
- Copyright (c) 2020 SolarWinds Worldwide, LLC
+ Copyright (c) 2021 SolarWinds Worldwide, LLC
```

Comment:
do we need to change any go file?
I think only updated in 2021 should have modified copyrights.

Reply:
I run sed according to your suggestions?

Reply:
I only suggested sed, didn't mention to run on all go files :) 
My point is that in prev company we updated copyright only in the case of change for some reason.
Can we make sure what are the rules in our company? Is there someone from legal etc we could ask about it?

Reply:
Well, this is a question more about how does your copyrights linter validate the code as I believe at that time your considerations were discussed and are kind of stated in its logic, am I right? So it only validates copyrights of file touched in the PR?

Reply:
I looked at the validator code and it looks so - the I'm gonna revert copyrights updates in files unaffected by the PR

Reply:
done...

Reply:
Well, that copyrights "linter" is red now https://app.circleci.com/pipelines/github/solarwinds/snap-plugin-lib/201/workflows/c06cf20f-f084-4345-9616-c99446f2a50d/jobs/554/parallel-runs/0/steps/0-103 will have to do a git history edit here...